docs/services/vision/segmentation.md

```diff
@@ -218,6 +219,81 @@ The following parameters are available for a `detector_3d_segmenter`.
 
 Click **Save config** and proceed to [test your segmenter](#test-your-segmenter).
 
+## Configure an `obstacles_depth` segmenter
+
+This segmenter model is for depth cameras, and is best for motion planning with transient obstacles.
+
+Use the segmenter to identify well separated objects above a flat plane.
+Configure the `with_geometries` attribute according to whether you've [configured the frame system](/services/frame-system/#configuration) to provide intrinsic parameters:
```

Comment:
@npentrel Edited this re. your review/suggestion to combine diff sections

Reply:
I still don't get this - you're saying here "wheter you've configured the frame system" and on 232 "start by configuring your frame system". Either you have already configured it or you start configuring it in 232. Or do you just configure part of the frame system subsequently. Also how do I choose whether I want to use a frame system. I think this should say something like "If you need to receive a point cloud, ..." "If you only need a single point, then you do not need ..."